README.md

```diff
@@ -43,7 +43,8 @@ bazel build -c opt //:nighthawk
 ```
 USAGE:
 
-bazel-bin/nighthawk_client  [--stats-flush-interval <uint32_t>]
+bazel-bin/nighthawk_client  [--response-latency-header-name <string>]
```

Comment:
Sorry for extra churn here.  Realized I gave you a misleading name (implies this would be response latency, which, while possible to do with this, is not the only usecase).

How do you feel about `--latency-response-header-name` (just inverting those two words).

Reply:
Sounds good, done in 594bbbec